src/v2/BaseLoginRouter.js

```diff
@@ -197,7 +197,7 @@ export default Router.extend({
         .catch(errorResp => {
           this.settings.unset('stateToken');
           this.settings.unset('useInteractionCodeFlow');
-          this.appState.trigger('remediationError', errorResp.error);
+          this.appState.trigger('remediationError', errorResp.error || errorResp);
```

Comment:
Error can be string

Reply:
Wow, when does the error come back as a string?

Reply:
Without mocking crypto there will be ConfigError with message `PKCE requires a modern browser with encryption support running in a secure context.\nThe current page is not being served with HTTPS protocol. PKCE requires secure HTTPS protocol
`